### PR TITLE
Show house card images instead of buttons

### PR DESCRIPTION
**Pull request by gereon77**
Implements part of #144

### CODE REVIEW

File: agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx [hunk 1]
```diff
@@ -51,6 +69,6 @@ export default class ChooseHouseCardComponent extends Component<GameStateCompone
             return [];
         }
 
-        return this.props.gameState.getChoosableCards(this.props.gameClient.authenticatedPlayer.house);
+        return this.props.gameState.getChoosableCards(this.props.gameClient.authenticatedPlayer.house).sort((a, b) => a.combatStrength - b.combatStrength);
```
Comment (Longwelwind) [inline agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx]: Not needed. The cards are ordered in `baseGameData.json`.

File: agot-bg-game-server/src/client/game-state-panel/SelectHouseCardComponent.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export default class SelectHouseCardComponent extends Component<GameStateCompone
                                         <Col xs="auto" key={hc.id}>
                                             <HouseCardComponent
                                                 houseCard={hc}
-                                                size="tiny"
+                                                size="small"
```
Comment (Longwelwind) [inline agot-bg-game-server/src/client/game-state-panel/SelectHouseCardComponent.tsx]: Why this change in particular?

File: agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx (hunk 1 above)
Comment (gereon77) [reply agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx]: @Longwelwind Without that it starts with the strongest card going down to the 0 card which is unusual. Most users sort their cards ascending.

File: agot-bg-game-server/src/client/game-state-panel/SelectHouseCardComponent.tsx (hunk 2 above)
Comment (gereon77) [reply agot-bg-game-server/src/client/game-state-panel/SelectHouseCardComponent.tsx]: @Longwelwind I played around with tiny and small for the chooseHcComp and realized that for selection of a house card small fits better here, especially on small screens. So I changed that here also to have a consistently look and feel when it comes to HC selection.

File: agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx (hunk 1 above)
Comment (Longwelwind) [reply agot-bg-game-server/src/client/game-state-panel/ChooseHouseCardComponent.tsx]: Nevermind, it's needed since the cards are ordered in a map and not in an array.

Ok then!

File: agot-bg-game-server/src/client/game-state-panel/SelectHouseCardComponent.tsx (hunk 2 above)
Comment (Longwelwind) [reply agot-bg-game-server/src/client/game-state-panel/SelectHouseCardComponent.tsx]: Ok.

**Comment by Longwelwind** [comment]
Rebase and I'll merge :)
